packages/graphiql/src/new-components/stories/Layout.stories.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
```

Comment:
for now, let's keep stories side by side with our components.

the only move we'd planned on making for this pr was to move `src/new-components` to `src/components/common`